GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -156,12 +145,22 @@ private string GetShortcutKeyDisplayString(Command cmd)
         public void DisplayDiffTab()
         {
             var revisions = _revisionGrid.GetSelectedRevisions();
+            SetDiffs(revisions);
+            if (DiffFiles.SelectedItem == null)
+            {
+                DiffFiles.SelectFirstVisibleItem();
+            }
+        }
+
+        private void SetDiffs(IReadOnlyList<GitRevision> revisions)
+        {
+            GitItemStatus oldDiffItem = DiffFiles.SelectedItem;
```

Comment:
I would have preferred that oldDiffItem was set only at explicit (clicked) file settings, but this is not so easy to achieve what I find